classes.js

```diff
@@ -0,0 +1,226 @@
+//Problem 1
+//a
+class Person {
+    constructor(firstName, lastName, middleName){
+        this.firstName = firstName
+        this.lastName = lastName
+        this.middleName = middleName
+    }
+    fullName(){
+        return `${this.firstName} ${this.middleName} ${this.lastName}`
+    }
+}
+let jovanni = new Person("Jovanni", "Luna", "David")
+let isaiah = new Person("Isaiah", "Collazo", "Ramon")
+console.log(jovanni.firstName)
+//b
+console.log(jovanni.fullName() + " " + isaiah.fullName())
+
+//Problem 2
+//a
+class Book{
+    constructor(title, author, rating){
+        this.title = title
+        this.author = author
+        this.rating = rating
+    }
+    isGood(){
+        if(this.rating >= 7){
+            return "true"
+        }
+    }
+}
+let nineteenEightyFour = new Book("1984", "George Orwell", 8)
+let thePowerOfPostiveThinking = new Book("The Power of Postive Thinking", "Norman Vicent Peale", 5)
+let ofMiceAndMen = new Book("Of Mice & Men", "John Steinbeck", 7)
+//b
+console.log(nineteenEightyFour.isGood())
+
+//Problem 4
+//a
+class Dog {
+    constructor(name, breed, mood, hungry){
+        this.name = name
+        this.breed = breed
+        this.mood = mood
+        this.hungry = hungry
+    }
+    playFetch(){
+        this.hungry = true
+        this.mood = "playful"
+        console.log("Ruff!")
+    }
+    feed(){
+        if(this.hungry === true){
```

Comment:
All good, but you can just check `if(this.hungry) ` since true would be only time the code in the if statement would execute.